config/imsim/ingest.py

```diff
@@ -0,0 +1,25 @@
+# This file is part of obs_lsst.
```

Comment:
No harm in including this, but it's already in the main branch of `obs_lsst`.

Reply:
Removed